Sources/MapboxMaps/Gestures/GestureManager.swift

```diff
@@ -255,12 +257,17 @@ extension GestureManager: GestureHandlerDelegate {
         return mapboxMap.cameraState.zoom
     }
 
-    internal func pinchScaleChanged(with newScale: CGFloat, andAnchor anchor: CGPoint) {
-        mapboxMap.setCamera(to: CameraOptions(anchor: anchor, zoom: newScale))
+    internal func pinchChanged(with zoom: CGFloat, anchor: CGPoint, previousAnchor: CGPoint) {
+        mapboxMap.dragStart(for: previousAnchor)
+        let dragOptions = mapboxMap.dragCameraOptions(from: previousAnchor, to: anchor)
+        mapboxMap.setCamera(to: dragOptions)
+        mapboxMap.dragEnd()
+
+        mapboxMap.setCamera(to: CameraOptions(anchor: anchor, zoom: zoom + mapboxMap.cameraState.zoom))
```

Comment:
I'm concerned that `zoom + mapboxMap.cameraState.zoom` will accumulate error. I'd like to explore a version of this that bases everything off of the zoom when the gesture began.